packages/api-gateway/src/graphql-common-utils.ts

```diff
@@ -0,0 +1,84 @@
+import { Connection, cursorToOffset, offsetToCursor } from 'graphql-relay';
+import type { QueryInput } from '@aloxide/demux';
+
+const DEFAULT_ITEM_PER_PAGE = 20;
+
+export function updateCursorToOffet(args: any): QueryInput {
+  const updatedArgs: QueryInput = Object.assign({}, args);
+  const { after, before } = updatedArgs;
+
+  if (typeof before === 'string' && !isNaN(cursorToOffset(before))) {
+    updatedArgs.before = cursorToOffset(before).toString();
+  } else delete updatedArgs.before;
+
+  if (typeof after === 'string' && !isNaN(cursorToOffset(after))) {
```

Comment:
Does it mean "if after is not a number, then delete it"? If yes, it's easier to read than "if after is not not a number then use it", double "not" make me think too much :)

Reply:
used another way that included in this commit https://github.com/lecle/aloxide/pull/126/commits/d93c71b8c56635972c082522651d5e7461ac5800